fixtures/american-dj/encore-lp12z-ip.json

```diff
@@ -0,0 +1,580 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Encore LP12Z IP",
+  "shortName": "encore-lp12z-ip",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["Nikita"],
+    "createDate": "2024-09-10",
+    "lastModifyDate": "2024-09-10"
+  },
+  "links": {
+    "manual": [
+      "https://d295jznhem2tn9.cloudfront.net/ItemRelatedFiles/13583/ADJ%20Encore%20LP12Z%20IP%20-%20User%20Manual%2002-22-2024.pdf",
+      "https://d295jznhem2tn9.cloudfront.net/ItemRelatedFiles/13583/ADJ%20Encore%20LP12Z%20IP%20-%20DMX%20Traits.pdf"
+    ],
+    "productPage": [
+      "https://www.adj.com/encore-lp12z-ip"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=f_LbX1cYPYw",
+      "https://www.youtube.com/shorts/7YWwZOtYoiw",
+      "https://www.youtube.com/shorts/4KwpsqBNHNc"
+    ]
+  },
+  "physical": {
+    "dimensions": [371, 316, 346],
+    "weight": 7.5,
+    "power": 227,
+    "DMXconnector": "3-pin XLR IP65",
+    "bulb": {
+      "type": "LED",
+      "lumens": 4400
+    },
+    "lens": {
+      "degreesMinMax": [5, 30]
+    }
+  },
+  "availableChannels": {
+    "Red": {
+      "fineChannelAliases": ["Red fine"],
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red",
+        "brightnessStart": "0%",
+        "brightnessEnd": "100%"
```

Comment:
Can you please remove `brightnessStart`/`brightnessEnd` for the color and dimmer channels? They are implicitly set anyway and don't help readability IMO.